lms/views/api/blackboard/files.py

```diff
@@ -0,0 +1,50 @@
+"""Proxy API views for files-related Blackboard API endpoints."""
+from pyramid.view import view_config, view_defaults
+
+from lms.views import helpers
+
+
+@view_defaults(permission="api", renderer="json")
+class BlackboardFilesAPIViews:
+    def __init__(self, request):
+        self.request = request
+
+    @view_config(request_method="GET", route_name="blackboard_api.courses.files.list")
+    def list_files(self):
+        """Return the list of files in the given course."""
+        self.request.find_service(name="oauth2_token").get()
```

Comment:
This `get()` is just so that it raises `NoOAuth2Token` if the fake access token hasn't been saved to the DB yet. The real code for this view would raise that when it really tried to do something. The fake code needs to artificially raise it to trigger the authorization flow